Gruntfile.js

```diff
@@ -167,59 +145,6 @@ module.exports = function (grunt) {
     'mocha:webserver',
   ]);
 
-  grunt.registerTask('release:refresh-pkgVersion', 'Refreshes GruntConfig.pkgVersion', function () {
-    grunt.config('pkgVersion', grunt.file.readJSON('package.json').version);
-    grunt.log.ok('pkgVersion updated');
-  });
-
-  grunt.registerTask('release:git-add-generated', 'Adds generated files to the git staging area', function () {
-    var done = this.async();
-    var generatedFiles = [
-      gruntConfig.dirs.common + '/lib/util/defaults.js',
-      gruntConfig.dirs.fragments + '/license.js',
-      'package.json',
-      'package-lock.json',
-      'README.md',
-      'test/support/browser_file_list.js',
-    ];
-    var cmd = 'git add -A ' + generatedFiles.join(' ');
-    grunt.log.ok('Executing ' + cmd);
-
-    require('child_process').exec(cmd, function (err, stdout, stderr) {
-      if (err) {
-        grunt.fatal('git add . -A failed with ' + stderr);
-      }
-      done();
-    });
-  });
-
-  grunt.registerTask('release:git-push', 'Pushes to git', execExternal('git push origin main --follow-tags'));
-
-  grunt.registerTask('release:ably-deploy', 'Deploys to ably CDN', function () {
-    var version = grunt.file.readJSON('package.json').version,
-      cmd = 'node scripts/cdn_deploy.js --skipCheckout --tag ' + version;
-    console.log('Publishing version ' + version + ' of the library to the CDN');
-    execExternal(cmd).call(this);
-  });
-
-  grunt.registerTask('release:deploy', 'Pushes a new release to github and then deploys to the Ably CDN', function () {
-    grunt.task.run(['release:git-push', 'release:ably-deploy']);
-  });
-
-  grunt.registerTask(
-    'release',
-    'Increments the version, regenerates, and makes a tagged commit. Run as "grunt release:type", where "type" is "major", "minor", "patch", "prepatch", etc.)',
-    function (versionType) {
-      grunt.task.run([
-        'bump-only:' + versionType,
-        'release:refresh-pkgVersion',
-        'all',
-        'release:git-add-generated',
-        'bump-commit',
-      ]);
-    },
-  );
-
```

Comment:
so all `release` grunt tasks are not used anymore?
probably they are not, there is no mention of grunt in `Release Process` in `CONTRIBUTING.md`

Reply:
yeah these tasks were for an old release process where we added generated build files to the git index, the steps in CONTRIBUTING.md is what we use now

Reply:
got it